src/components/cms/BrowseSkill/BrowseSkill.js

```diff
@@ -410,12 +409,14 @@ class BrowseSkill extends React.Component {
       };
     }
     this.props.actions.getSkills(payload);
+    scrollAnimation(document.documentElement, 0, 200, 'vertical');
   };
 
   loadMetricsSkills = () => {
     this.props.actions.getMetricsSkills({
       languageValue: this.props.languageValue,
     });
+    scrollAnimation(document.documentElement, 0, 200, 'vertical');
```

Comment:
How about implementing in the `withTracker` HOC in your other PR?

Reply:
Oh, nice idea. But we would require inline scroll calls as well for metrics change on browseskill

Reply:
To handle that, we should actually add the url param on metric change to the url as well.

Reply:
Okay working on it, created issue https://github.com/fossasia/susi.ai/issues/2651